Makefile

```diff
@@ -118,6 +118,10 @@ verify: .init .generate_files
 	  ($(DOCKER) golint --set_exit_status $$i && \
 	   $(DOCKER) go vet $$i )|| exit $$? ; \
 	done || false
+	@echo Running repo-infra verify scripts
+	$(DOCKER) vendor/github.com/kubernetes/repo-infra/verify/verify-boilerplate.sh --rootdir=. | grep -v zz_generated > .out 2>&1 || true
```

Comment:
The generated files should actually be added with a correct boilerplate, but I understand this for now.